actionview/lib/action_view/base.rb

```diff
@@ -190,6 +190,14 @@ def with_empty_template_cache # :nodoc:
           # correctly.
           define_method(:compiled_method_container)           { subclass }
           define_singleton_method(:compiled_method_container) { subclass }
+
+          def self.name
+            superclass.name
+          end
+
+          def inspect
+            "#<#{self.class.name}>"
+          end
```

Comment:
Replacing the anonymous subclass' name with `ActionView::Base` makes sense, but I don't see the value in stripping out the object ID here. I suspect this could actually be _more_ confusing, since the format doesn't match what you're used to if you've ever seen a NoMethodError before.

Reply:
Yes, I agree and will fix this.

Reply:
@eugeneius I've added the `object_id` and tried to keep the same formatting:
`#<ActionView::Base:0x0000000000cda0>`
I've copied the formatting of the object_id from rspec:
https://github.com/rspec/rspec-support/blob/main/lib/rspec/support/object_formatter.rb#L215

Reply:
Perfect, thanks @p8!